src/js/actions/create-event.js

```diff
@@ -213,7 +213,7 @@ export function removeInvitee (invitee, index) {
 HYDRATE EDIT EVENT ACTIONS
 ********/
 
-export function hydrateEditEvent (event) {
+export function hydrateCreateEvent (event) {
     return {
         type: HYDRATE_EDIT_EVENT,
```

Comment:
@sohilpandya you need to change the actual action name as well, from `HYDRATE_EDIT_EVENT` to `HYDRATE_CREATE_EVENT`, in the action creator files, the reducers and their respective tests